.github/ISSUE_TEMPLATE/06_localization_data_issue.yml

```diff
@@ -0,0 +1,104 @@
+        * For any local user-facing (most often GUI-based) application, you **MUST** respect and use the culture of the user.
+        * When practicable, you _SHOULD_ consider using and enabling format overrides.  On Windows this can be achieved just by calling the proper [`CultureInfo()` constructor](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.-ctor?view=net-8.0#system-globalization-cultureinfo-ctor(system-string-system-boolean)).  Other operating systems or web applications may require custom code.
+        * For applications that are distributed, allow zero-downtime updates (eg, Azure Function Applications), or similar you **MUST PLAN** for situations where there may be multiple instances of the application with different format or other culture data (including timezone rules).
+        * For any element being used for interchange (eg, a field in a JSON document in an API call) or storage, you **MUST** use the [`InvariantCulture`](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-8.0); additionally, in many cases you _SHOULD_ either use a roundtrip format (such as the [`"O"`, `"o"` date-time format specifier](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier)), or explicitly specify a format string.  Although the format strings for the invariant culture are stable and unlikely to change, specifying an explicit format string helps to clarify the intent of your code.
+          * For date/time elements, consider the advice and observations in Jon Skeet's post [_Storing UTC is not a silver bullet_](https://codeblog.jonskeet.uk/2019/03/27/storing-utc-is-not-a-silver-bullet/).
+        * You **MUST NOT** [consider culture data to be stable](https://learn.microsoft.com/en-us/archive/blogs/shawnste/culture-data-shouldnt-be-considered-stable-except-for-invariant).  This applies to both format strings and other culture data like timezone rules.
+        * To make dependencies more explicit and testing potentially easier and more parallelizable, you _SHOULD CONSIDER_ explicitly passing culture-relevant settings, such as `CultureInfo` parameters, to methods that perform formatting or `TimeZoneInfo` to methods that work with dates and times.  You **MUST** use [`TimeProvider`](https://learn.microsoft.com/en-us/dotnet/api/system.timeprovider?view=net-8.0) or a similar type when retrieving the time.
```

Comment:
Maybe pointing at the doc https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings will be helpful.